docrane/container.py

```diff
@@ -65,6 +73,19 @@ def ensure_running(self, force_restart=False):
                     LOG.warning("Image ID has changed. Restarting container.")
                 break
 
+        try:
+            image_found = util.pull_image(
```

Comment:
I think I like the in-util exception. It's meant to pull an image, and should error if it can't.